interfaces/Portalicious/src/app/components/page-layout/components/registration-header/add-note-form/add-note-form.component.html

```diff
@@ -0,0 +1,33 @@
+<app-form-sidebar
+  [(visible)]="formVisible"
+  [formGroup]="formGroup"
+  [mutation]="addNoteMutation"
+  formTitle="Add note"
+  i18n-formTitle="@@add-note"
+>
+  <p
+    i18n
+    class="font-semibold"
+  >
+    You are about to add a note to {{ registrationName() }}'s profile.
+  </p>
+  <p
+    i18n
+    class="mb-8"
+  >
+    The note will be visible to other 121 users who are permitted to view
+    personal profiles.
+  </p>
+  <app-form-field-wrapper
+    label="Note"
+    i18n-label
+    [errorMessage]="formFieldErrors()('note')"
+  >
+    <textarea
+      rows="5"
+      pInputTextarea
+      formControlName="note"
+      autocomplete="off"
+    ></textarea>
```

Comment:
No hint-text about the keyboard-shortcut here? (like in the screenshot?)

Reply:
I missed it in the screenshot! Adding it now :)